compiler/modules/types/error.bal

```diff
@@ -1,7 +1,7 @@
 // Implementation specific to basic type error.
 
 public function errorDetail(SemType detail) returns SemType {
-    SubtypeData sd = subtypeData(detail, UT_MAPPING_RO);
+    SubtypeData sd = subtypeData(detail, BT_MAPPING);
```

Comment:
Let's introduce a convention for a comment to mark a place where will have to fix something for READONLY